xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-macro/src/main/resources/ApplicationResources.properties

```
@@ -106,4 +106,13 @@ livedata.panel.sort.direction.descending=Descending
 livedata.panel.sort.add=Add sort
 livedata.panel.sort.delete=Delete sort
 
-livedata.displayer.link.noValue=(no value)
\ No newline at end of file
+livedata.displayer.link.noValue=(no value)
+
+livedata.displayer.boolean.true=True
+livedata.displayer.boolean.false=False
+
+livedata.displayer.xObjectProperty.missingDocumentName.errorMessage=Can't save an XObject property with a missing document name.
+livedata.displayer.xObjectProperty.failedToRetrieveField.errorMessage=Failed to retrieve the {mode} field.
```

The problem is that contributors will translate also "mode" so best is to use:

```suggestion
livedata.displayer.xObjectProperty.failedToRetrieveField.errorMessage=Failed to retrieve the {0} field.
```